app/_src/policies/meshmetric.md

```diff
@@ -0,0 +1,284 @@
+
+#### Running multiple prometheus instances
+
+If you need to run multiple instances of Prometheus and want to target different set of DPPs you can do this by using Client ID setting on both `MeshMetric` (`clientId`) and [Prometheus configuration](https://github.com/prometheus/prometheus/pull/13278/files#diff-17f1012e0c2fbd9bcd8dff3c23b18ff4b6676eef3beca6f8a3e72e6a36633334R2233) (`client_id`).
+
+{% warning %}
+To use `clientId` setting you need to be running at least Prometheus `2.49.0`.
+{% endwarning %}
+
+Example configurations differentiated by `prometheus` tag:
+
+{% policy_yaml first %}
+```yaml
+type: MeshMetrics
+name: prometheus-one
+mesh: default
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      prometheus: "one"
+    backends:
+      - type: Prometheus
+        prometheus: 
+          clientId: "prometheus-one" 
+          port: 5670
+          path: /metrics
+```
+{% endpolicy_yaml %}
+
+{% policy_yaml second %}
+```yaml
+type: MeshMetrics
+name: prometheus-two
+mesh: default
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      prometheus: "two"
+    backends:
+      - type: Prometheus
+        prometheus: 
+          clientId: "prometheus-two" 
+          port: 5670
+          path: /metrics
+```
+{% endpolicy_yaml %}
+
+And the Prometheus configurations:
+
+```yaml
+scrape_configs:
+  - job_name: 'kuma-dataplanes'
+    # ...
+    kuma_sd_configs:
+    - server: http://localhost:5676
+      refresh_interval: 60s # different from prometheus-two
+      client_id: "prometheus-one"
+```
+
+```yaml
+scrape_configs:
+  - job_name: 'kuma-dataplanes'
+    # ...
+    kuma_sd_configs:
+      - server: http://localhost:5676
+        refresh_interval: 20s # different from prometheus-one
+        client_id: "prometheus-two"
+```
+
+## Examples
+
+### With custom port, path, clientId, application aggregation and service override
+
+The first policy defines a default `MeshMetric` policy for the `default` mesh.
+The second policy creates an override for workloads tagged with `framework: example-web-framework`.
+That web framework exposes metrics under `/metrics/prometheus` and port 8888.
+
+{% policy_yaml customone %}
+```yaml
+type: MeshMetric
+mesh: default
+name: metrics-default
+targetRef:
+  kind: Mesh
+default:
+  sidecar:
+    usedOnly: true
+  backends:
+    - type: Prometheus
+      prometheus:
+        clientId: main-backend 
+        port: 5670
+        path: /metrics
+        tls:
+          mode: "ProvidedTLS"
+```
+{% endpolicy_yaml %}
+
+{% policy_yaml customtwo %}
+```yaml
+type: MeshMetric
+mesh: default
+name: metrics-for-mesh-service
+targetRef:
+  kind: MeshSubset
+  tags:
+    framework: "example-web-framework"
+default:
+  applications:
+    - path: "/metrics/prometheus"
+      port: 8888
+```
+{% endpolicy_yaml %}
+
+## All policy options
+
+{% json_schema MeshMetrics %}
```

Comment:
Shouldn't it be `MeshMetric`?

```suggestion
{% json_schema MeshMetric %}
```

Reply:
The schema rendered - https://deploy-preview-1578--kuma.netlify.app/docs/dev/policies/meshmetric/#all-policy-options so I don't think so, this one needs to be plural.

Reply:
Makes sense!